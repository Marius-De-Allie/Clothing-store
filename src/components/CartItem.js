import React from 'react';
// styles.
import '../styles/cartItem.scss';

const CartItem = ({ item }) => (

    <div className="cart-item">
        <img src={item.imageUrl} alt={item.name} />
        <div className="item-details">
            <span className="name">{item.name}</span>
            <span className="price">{item.qantity} x ${item.price}</span>
        </div>
    </div>
);

export default CartItem;