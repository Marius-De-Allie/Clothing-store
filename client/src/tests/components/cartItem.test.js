import React from 'react';
import { shallow } from 'enzyme';
import CartItem from '../../components/CartItem';

test('CartItems component renders properly', () => {
    const wrapper = shallow(<CartItem item={{imageUrl: '', name:'', quantity: 0, price: 0}} />);
    expect(wrapper).toMatchSnapshot();
});