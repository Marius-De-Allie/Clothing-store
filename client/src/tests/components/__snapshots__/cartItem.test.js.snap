// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItems component renders properly 1`] = `ShallowWrapper {}`;
